### ISSUE TITLE
ES 7+ compatibility

### ISSUE THREAD

**Issue by usego** [braedon/prometheus-es-exporter#41]
I've upgraded ES to 7.2 and noticed that mine metrics against ES failed. After some research I found that metric if failing with:

```
[2019-07-19 17:50:56,967] root.ERROR MainThread Error while running scheduled job.
Traceback (most recent call last):
  File "/usr/src/app/prometheus_es_exporter/__init__.py", line 210, in scheduled_run
    func()
  File "/usr/src/app/prometheus_es_exporter/__init__.py", line 121, in run_query
    update_gauges(metrics)
  File "/usr/src/app/prometheus_es_exporter/__init__.py", line 89, in update_gauges
    gauge.set(value)
  File "/usr/local/lib/python3.6/site-packages/prometheus_client/metrics.py", line 340, in set
    self._value.set(float(value))
TypeError: float() argument must be a string or a number, not 'dict'
```

I think the reason is that hits output is [changed in ES7](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#hits-total-now-object-search-response) which breaks things. So or additional parameter rest_total_hits_as_int=true should be added to search queries URL, or some ES7 support should be introduced as rest_total_hits_as_int parameter will be removed in ES8.  

Please advise how this issue could be fixed

**Comment by usego** [comment]
Adding ['value'] to parse_response fixes an issue for me:

result.append((metric + ['hits'], {}, response['hits']['total']['value']))

Sorry, could not provide PR as mine python skills are far from be able to do this in normal way to provide backward compatibility with ES6.

**Comment by braedon** [comment]
Thanks for reporting this @usego, I'll have a play and see if I can get the parsing to handle both cases automatically.

**Comment by braedon** [comment]
Fixed in [0.5.2](https://github.com/braedon/prometheus-es-exporter/releases/tag/0.5.2)
